includes/forms.inc

```diff
@@ -41,8 +41,32 @@ function islandora_ip_embargo_settings_form($form, &$form_state) {
       0
     ),
   );
+  $form['islandora_ip_embargo_overlay_text'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Embargoed item overlay text'),
+    '#description' => t('Value for overlay text on embargoed items.'),
+    '#default_value' => variable_get(
+      'islandora_ip_embargo_overlay_text',
+      'EMBARGOED'
+    ),
+  );
+  $form['islandora_ip_embargo_overlay_text_color'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Embargoed item overlay text color'),
+    '#description' => t('Value for overlay text color in hexideciamal format (E.g. FF0000).'),
+    '#maxlength' => 6,
+    '#default_value' => variable_get(
+      'islandora_ip_embargo_overlay_text_color',
+      'FF0000'
+    ),
+  );
```

Comment:
May just want to roll an (element_validate)[https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7.x#element_validate] that uses Drupal core's Color module (color_valid_hexadecimal_string)[https://api.drupal.org/api/drupal/modules%21color%21color.module/function/color_valid_hexadecimal_string/7.x] for completeness sake.